src/osd/PG.cc

```diff
@@ -818,7 +818,6 @@ void PG::check_past_interval_bounds() const
       derr << info.pgid << " required past_interval bounds are"
 	   << " not empty [" << rpib << ") but past_intervals "
 	   << past_intervals << " is empty" << dendl;
-      assert(!past_intervals.empty());
```

Comment:
I think this assert is intentional made to catch some unexpected/bad behaviour..

Reply:
In that case, it may probably make more sense to leave some comment.

Reply:
the derr and error() lines above are the comment